lib/AST/FeatureSet.cpp

```diff
@@ -185,6 +185,10 @@ static bool usesFeatureSendingArgsAndResults(Decl *decl) {
   return false;
 }
 
+static bool usesFeatureExtern(Decl *decl) {
+  return decl->getAttrs().hasAttribute<ExternAttr>();
```

Comment:
This is not needed, we don't need to support compilers that can't parse the `@_extern` attribute anymore.